.circleci/config.yml

```diff
@@ -177,6 +190,24 @@ jobs:
             fi
           when: on_fail
 
+  upload_operator:
+    <<: *python_container_config
+    steps:
+      - attach_workspace:
+          at: snyk-operator
+      - run:
+          name: Install operator-courier
+          command: pip3 install operator-courier
+      - run:
+          name: Upload Operator to Quay
+          command: |
+            export QUAY_TOKEN=$(curl -H "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d "{\"user\": {\"username\": \"${QUAY_USERNAME}\", \"password\": \"${QUAY_PASSWORD}\"}}" | jq -r .token)
```

Comment:
consider using a script (Python or Shell) to do this, I'm concerned we're loading too many things on our circleci config file

Reply:
I think it makes sense to extract it out to some script so it can be run locally, but what is your bigger vision here? To have every CI step be in its own separate script?

Reply:
I think we should add a general task to clean our circleci config, it is getting huge